agent-install/test/README.md

```diff
@@ -0,0 +1,57 @@
+# Automated tests
+
+## Overview
+
+This script:
+
+* Gets specified packages distributives
+* Verifies installation and functionality of the `agent-install.sh` script
+* Can be used for cluster heatchcheck
+
+## Requirements
+
+Currently supported OS and architectures:
+
+* Ubuntu 16.04, 18.04
+  * amd64
+
+## Quick start
+
+1. Get the configuration file `agent-install.cfg` and certificate `agent-install.crt` files for your environment, and copy them in `config` subdirectory.
+2. Get the configuration file `agent-install.cfg` and certificate `agent-install.crt` files for second environment (the script will test switching between you primary environment and one you specifed as "switch" or "secondary" environment), and copy them in `config/switch` subdirectory.
```

Comment:
Please add some explanation to the Overview section which describes the switching feature.